fastly/utils.py

```diff
@@ -1,22 +1,20 @@
 import requests
 
+from urllib.parse import urljoin
+
 from django.conf import settings
 
 
-def purge_url(path):
+def purge_url(*paths):
```

Comment:
What do you think about renaming this to `purge_urls()`?

Reply:
Done